wls-gui-admintool/tests/components/common/BaseAutocompleteWahltag.spec.ts

```diff
@@ -10,24 +12,36 @@ import BaseAutocompleteWahltag from "@/components/common/BaseAutocompleteWahltag
 import pinia from "@/plugins/pinia";
 import { getSnapshotFilename } from "../../utils/testutils.ts";
```

Comment:
```suggestion
import { getSnapshotFilename } from "@tests/utils/testutils.ts";
```

es gibt noch mehr files aus anderen prs, wo das mit angepasst werden könnte. da der alias hier im pr eingeführt wurde und das nur 3 files sind könnte man das hier noch mit umsetzen: 
![grafik](https://github.com/user-attachments/assets/a59fd8cb-7453-4ab5-83ed-ae0792904ec2)

Reply:
In wls-gui-wahllokalsystem kann ich es nicht anpassen da die Vorraussetzungen fehlen. Das mache ich in einem separatem PR